tools/crm_resource.c

```diff
@@ -1648,6 +1648,20 @@ main(int argc, char **argv)
             exit_code = pcmk_rc2exitc(rc);
             goto done;
         }
+
+        /* If user supplied a node name, check whether it exists.
+         * Commands that don't require scheduler data ignore the node argument.
+         */
+        if (options.host_uname != NULL) {
```

Comment:
@clumens This commit causes the "Try to move an instance of a cloned resource" test to fail because `node1` is not found. So we return `CRM_EX_NOSUCH` instead of the expected `CRM_EX_INVALID_PARAM`.

I'm not sure why `node1` is not found. It looks like the test setup is supposed to create it:
```
        basic_tests_setup = [
            "crm_attribute -n no-quorum-policy -v ignore",
            "crm_simulate --live-check --in-place --node-up=node1"
        ]
```

Reply:
Wait, maybe that only applies to `basic_tests` and not to `basic_tests2`, where this particular test lives.